int32 lane_serial
uint32 contour_serial
float64[3] params
geometry_msgs/Point32[] points
